drivers/video/ov2640.c

```diff
@@ -1077,8 +1095,14 @@ static int ov2640_init_0(const struct device *dev)
 	}
 #endif
 
-	uint32_t i2c_cfg = I2C_MODE_CONTROLLER |
-					I2C_SPEED_SET(I2C_SPEED_STANDARD);
+#if DT_INST_NODE_HAS_PROP(0, pwdn_gpios)
```

Comment:
The problem with this is that it only checks for the instance `0` (the first one).
If there are 2 sensors and only the 2nd has `pwdn_gpios`, then this would not work.

You might need something like this:

```suggestion
#if DT_ANY_INST_HAS_PROP_STATUS_OKAY(pwdn_gpios)
	if (cfg->pwdn_gpios.port != NULL) {
```

The `#if` helps making `cfg->pwdn_gpios` compile.

The `if ()` helps detecting the per-instance presence/absence of a GPIO port.

You might need to replicate this for each occurence of `DT_INST_NODE_HAS_PROP(0, pwdn_gpios)`.

Would that be ok?
Thank you!